ci/rpm7/main/build

```diff
@@ -0,0 +1,344 @@
+#!/bin/bash
+
+path='/opt/fiware-orion'
+url_src='https://github.com/telefonicaid/fiware-orion'
+url_dst='https://nexus.lab.fiware.org/repository/el'
+releasever=7
+basearch='x86_64'
+
+function _usage()
+{
+  echo -n "Usage: build [options]
+  Options:
+    -H   --show          show help for functional test
+    -b   --branch        choose branch/tag to build, if not - source from /opt/fiware-orion will be used
+    -s   --source        repo to clone, if not - default from https://github.com/telefonicaid/fiware-orion
+    -M   --make          cmake/make
+    -S   --split         split make/cmake to stages
+    -I   --install       make install
+    -E   --execute       run (rerun) test stand with 2 orions
+    -u   --unit          run unit test
+    -f   --functional    run functional test
+    -n   --nightly       build rpm nightly
+    -r   --release       build rpm release
+    -t   --testing       build rpm testing
+    -U   --upload        upload rpm, REPO_USER and REPO_PASSWORD ENV variables should be provided
+    -F   --fix           fix for jenkins
+    -D   --db            start mongodb
+
+  Examples:
+    build -MIDufb master - clone from master, make, make install, run mongo, execute unit and functional tests
+    build -MIDFSf - get source from mounted folder, make, make install special for functional tests, execute FIX, functional test
+"
+  exit 0
+}
+
+function _fix_jenkins()
+{
+    echo "Builder: fix jenkins"
+    cp ${path}/test/functionalTest/testHarness.sh /tmp/builder/bu/testHarness.sh
+
```

Comment:
A FIXME comment should be added here, explaing that these test are skipped in Travis checks (and, if possible, all the information we have about the reason, to help people to solve the problem in the future).

Reply:
This problem is quite bigger. Some of tests skipped in testHarness itself, and with this function i'm only extend list for CI env. Maybe it's a better solution to move list of all skipped tests from testHarness to this function, or if not, i will add some kind of warning or additional message.

Reply:
Just to check... you finally solved this and the functional test that were failing are now fixed? In that case, please conclude this thread with NTC.

Reply:
2 of 3. cases/0000_ipv6_support/ipv4_ipv6_both.test should be disable for jenkins and travis. this test can be disabled but -F parameter for command build. As u can see this parameter is on in travis.yml

Reply:
NTC